langchain4j-anthropic/src/main/java/dev/langchain4j/model/anthropic/AnthropicChatModel.java

```diff
@@ -84,6 +88,8 @@ public class AnthropicChatModel implements ChatLanguageModel {
      * @param maxRetries    The maximum number of retries for API requests. Default: 3
      * @param logRequests   Whether to log the content of API requests using SLF4J. Default: false
      * @param logResponses  Whether to log the content of API responses using SLF4J. Default: false
+     * @param cacheSystemMessage  If true, it should add cache control to all system messages. Default: false
```

Comment:
```suggestion
     * @param cacheSystemMessages  If true, it should add cache control to all system messages. Default: false
```
Should it be plural?

Reply:
It should be changed in all other places as well